plugins/main/public/controllers/management/components/management/statistics/statistics-overview.js

```diff
@@ -240,41 +207,14 @@ export class WzStatisticsOverview extends Component {
             </EuiFlexItem>
           </EuiFlexGroup>
           <EuiSpacer size={'m'} />
-          {
-            <>
-              {this.state.selectedTabId === 'remoted' &&
-                !this.state.loadingNode && (
-                  <div>
-                    <EuiSpacer size={'m'} />
-                    <EuiCallOut
-                      title={this.info[this.state.selectedTabId]}
-                      iconType='iInCircle'
-                    />
```

Comment:
**nitpick**: the message displayed on the callout depends on the selected tab. The messages are defined on https://github.com/wazuh/wazuh-dashboard-plugins/pull/6542/files#diff-d25436682ccb826daef21db2a5c0cfb1b019e7c906b16bb9401d2aa0f31a8acfR72-R77

Reply:
Fixed information message based on active tab in `dashboardTabsPanels`